app/scripts/modules/core/application/config/trafficGuard/trafficGuardConfig.component.html

```diff
@@ -0,0 +1,71 @@
+<div class="panel panel-default">
+  <div class="panel-body form-inline">
+    <div class="row">
+      <div class="col-md-12">
+        <p>Traffic Guards allow you to specify critical clusters that should always have active instances. If a user
+          or process tries to delete, disable, or resize the server group, Spinnaker will verify the operation will not
+          leave the cluster with no active instances, and fail the operation if it would.</p>
+      </div>
+    </div>
+    <div class="row" ng-if="$ctrl.initializing">
+      <h4 class="text-center"><span class="glyphicon glyphicon-asterisk glyphicon-spinning"></span></h4>
+    </div>
+    <div class="row" ng-if="!$ctrl.initializing">
+      <div class="col-md-10 col-md-offset-1">
+        <table class="table table-condensed">
+          <thead>
+            <tr>
+              <th>Account</th>
+              <th>Region <help-field key="trafficGuard.region"></help-field></th>
```

Comment:
Agreed.  Does it make sense to have a NF override for this or make it configurable?